src/View/Cell.php

```diff
@@ -176,6 +176,17 @@ public function render($template = null)
         }
 
         $render = function () use ($template) {
+            try {
+                $reflect = new ReflectionMethod($this, $this->action);
+                $reflect->invokeArgs($this, $this->args);
```

Comment:
> If you had not refactored the cache to use remember() (meaning the render method would have been in that form (this is your first implementation of the cache cell), I would have agreed and not made the change.

Right, my memory failed me. What I said above was true before I re-factored to use Cache::remember() :)